syntax = "proto3";

package PeoplePackage;

service PeopleService {
    rpc GetData(PeopleId) returns (PeopleData) {}
    rpc GetListData(PeopleIdList) returns (stream PeopleData) {}
}

message PeopleIdList {
    repeated int32 id = 1;
}

message PeopleData {
    string name = 1;
    int32 age = 2;
    string address = 3;
}

message PeopleId {
    int32 id = 1;
}

